p4-16/psa/PSA.mdk

```diff
@@ -293,12 +293,15 @@ few packets dropped for these reasons (perhaps only one).
 ## PSA type definitions {#sec-psa-type-definitions}
 
 Each PSA implementation will have specific bit widths for the
-following types.  These widths should be defined in the target
-specific implementation of the `psa.p4` include file.
+following types in the data plane.  These widths should be defined in
+the target specific `psa.p4` include file.  They are expected to
+differ from one PSA implementation to another.
 
-The P4 Runtime API uses 64-bit values to hold values of these types.
-All PSA implementations must use sizes for these types no wider than
-the corresponding types with `InHeader_t` in their names shown below.
+For each one of these types, the P4 Runtime API uses values with the
+bit width of the corresponding `InHeader_t` type below to hold a value
+of that type, independent of the target.  All PSA implementations must
+use sizes for these types no wider than the corresponding types with
+`InHeader_t` in their names shown below.
```

Comment:
The original PSA v1.0 spec text here was written at least a month ago, when my understanding at the time from talking to Samar Abdi was the the controller/agent communication for all values of these types would be 64 bits wide.  Recent P4-API working group discussion makes it sound like values that are the same width as the corresponding `InHeader` types are perhaps preferable for this purpose.